backend/.env

```diff
@@ -1 +1,3 @@
-userDB_URL=""
\ No newline at end of file
+userDB_URL=""
+CLIENT_ID = 240698735710-tmaskolmt3u9qlp9552u33fpee8iu6u6.apps.googleusercontent.com
+CLIENT_SECRET = 891KfiFrYubCbf60guZPWmjd
```

Comment:
Hi, we are going to try to keep secrets off of our repository if possible for security reasons! :) Does google have an easy way for you to get a new secret? You can then share the new secret with all the devs on our private slack!